### DISCUSSION TITLE
What is cpress and kpress?

### DISCUSSION THREAD

**Discussion by muziker** [FluidSynth/fluidsynth#863] [Q&A] [answered]
What do cpress and kpress refer to? kpress is probably key pressure, but what does cpress do?

**Comment by mawe42** [comment, marked as answer]
I assume you are talking about the arguments to the `router_begin` commands.  `kpress` is key pressure (aka polyphonic aftertouch) as you have guessed. `cpress` is channel pressure, aka channel aftertouch.
